sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/AesCbc.cs

```diff
@@ -15,10 +15,12 @@ internal class AesCbc
         public static readonly AesCbc Aes128Cbc = new AesCbc("A128CBC", 128, PaddingMode.Zeros);
         public static readonly AesCbc Aes192Cbc = new AesCbc("A192CBC", 192, PaddingMode.Zeros);
         public static readonly AesCbc Aes256Cbc = new AesCbc("A256CBC", 256, PaddingMode.Zeros);
+        public static readonly AesCbc CKM_AES_KEY_WRAP = new AesCbc("CKM_AES_KEY_WRAP", 256, PaddingMode.Zeros);
```

Comment:
The names for the new members are in a different format than the existing ones.   We should update them so they follow the existing patterns.